### PR TITLE
Remove TextEncoder polyfill (#779)

### PR DESCRIPTION
**Pull request by maxammann**
TextEncoder is supported in the all browsers for a long time and also in node.js. For some node.js versions it is only available through the util package.

We do not need to support browsers like IE11 because they do not support webassembly.

This PR works on all browsers which support TextEncoding. It also works on all node versions until v8.3.0

## Checklist

- [x] Change is covered by automated tests
- [x] Benchmark results are attached (if applicable)
- [x] The [coding guidelines] are followed
- [x] Public API has proper documentation
- [x] Example projects and code samples are up-to-date (in case of API changes)
- [x] Changelog is updated (in case of notable or breaking changes)

[coding guidelines]: https://github.com/cossacklabs/themis/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Review by vixentael — APPROVED** [review]
Thank you @maxammann!

Right now, I have no idea why we used `fastestsmallesttextencoderdecoder` before, maybe @ilammy can recall. 

Looks like all tests are passing, will merge after @ilammy approve! 💪

**Comment by maxammann** [comment]
My PR basically replaces that polyfill with the node utils TextEncoder.

That should reduce the bundle size for browsers and still work on node <= 10

**Review by ilammy — APPROVED** [review]
It's nice to have people who actually know what they are doing to take a look at the code 👍

At the time of #616 I probably was not aware of `TextEncoder` being available in `utils`, hence this awkward dependency. And I pretty much welcome reducing the dependency scope.

**Pull request merged**
